src/App.tsx

```diff
@@ -1,8 +1,98 @@
 import React from 'react';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+import { Todo, User, PreparedTodo } from './components/Interfaces/Interfaces';
+import { TodoList } from './components/TodoList/TodoList';
+import { getUsers, getTodos } from './api';
+import { Buttons } from './components/Buttonns/Buttons';
+
+interface State {
+  todos: PreparedTodo[];
+  isLoading: boolean;
+}
+
+class App extends React.Component<{}, State> {
+  state = {
+    todos: [],
+    isLoading: false,
+  };
+
+  loadTodos = async () => {
+    this.setState({
+      isLoading: true,
+    });
+
+    const todos: Todo[] = await getTodos();
+    const users: User[] = await getUsers();
```

Comment:
опиши что отдают функции, здесь типы не нужны, они будут унаследованы
```suggestion
    const todos = await getTodos();
    const users = await getUsers();
```